airbyte-integrations/bases/base-java/Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /airbyte
 ADD https://dtdg.co/latest-java-tracer dd-java-agent.jar
 
 COPY javabase.sh .
+COPY run_with_normalization.sh .
```

Comment:
@edgao I think this should be dynamically added by Dagger on connectors supporting destination and not declared in the base java image. To not make your interaction on this topic blocked by Dagger implementation I suggest you keep this script in this folder but directly copy it to the bigquery destination.

Reply:
hm... are you thinking of a gradle copy task to get the script, and then a docker COPY? that feels kind of heavyweight

at the very least, a naive `COPY ../../bases/base-java/run_with_normalization.sh .` doesn't work, iirc docker copies files to a separate build context

Reply:
Ok! Feel free to keep it like it but on the Dagger side I think this should be only added on top of the java base if we need to package normalization.

Reply:
sounds good. I'll tag you on the dagger pr later this week to make sure I have the right idea